x/blob/client/testutil/integration_test.go

```diff
@@ -62,8 +64,8 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 	require := s.Require()
 	val := s.network.Validators[0]
 	hexNamespace := hex.EncodeToString(appns.RandomBlobNamespaceID())
+	invalidNamespaceID := hex.EncodeToString(bytes.Repeat([]byte{0}, 8)) // invalid because ID is expected to be 10 bytes
```

Comment:
> // invalid because ID is expected to be 10 bytes

@rootulp is this comment true?

Reply:
Yep! As a convenience, the CLI accepts a 10 byte namespace ID b/c that's the only usable portion for namespace version `0`.